packages/server/src/api/controllers/pools.ts

```diff
@@ -6,34 +6,28 @@ import {
   startOfHour,
   subWeeks,
 } from 'date-fns';
-import { Request, Response, Router } from 'express';
+import { Request, Router } from 'express';
 
-import { dayMs } from 'util/date'
 import { HTTPError } from 'api/util/errors';
 import { isValidEthAddress } from 'util/eth';
-import cacheMiddleware from 'api/middlewares/cache';
+import catchAsyncRoute from 'api/util/catch-async-route';
 import UV3Fetchers from 'services/uniswap-v3/fetchers';
-import wrapRequest from 'api/util/wrap-request';
 
 // TODO: move this somewhere else, maybe to the fetchers manager.
 // Will also need to namespace by network
-import { wrapWithCache, keepCachePopulated } from 'util/redis-data-cache';
```

Comment:
just some cleanup until we introduce cache namespacing with the network switching work